api/app/controllers/review.controller.ts

```diff
@@ -16,6 +16,28 @@ import {
   buildIsInActiveCondition,
   buildIsPendingCondition,
 } from "../utils/review.inference";
+import { parseMdToJSON } from "../helpers/transcript";
+import axios from "axios";
+import { BaseParsedMdContent, TranscriptAttributes } from "../types/transcript";
+
+const transcriptWrapper = async (transcript: TranscriptAttributes) => {
```

Comment:
Can you please comment here to better explain what this method does. I've read it a few times and it is still not clear to me.

Reply:
sure